catalog/core/catalog-core-api-impl/src/test/groovy/ddf/catalog/util/impl/ResultIterableSpec.groovy

```diff
@@ -28,10 +29,12 @@ import spock.lang.Unroll
 import static ddf.catalog.util.impl.ResultIterable.resultIterable
 import static java.util.stream.Collectors.toList
 
-class QueryResultPaginatorSpec extends Specification {
+class ResultIterableSpec extends Specification {
```

Comment:
❓ Do we have all the tests required to cover the code related to the new `actualResultSize` property, and all the other possible scenarios, e.g., empty result list returned by more results are still available?

Reply:
added